news/2024/2024-09-14-new-featured-artist-tomspicy.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: "New Featured Artist: tomspicy"
+date: 2024-09-14 19:00:00 +0000
+---
+
+Send only your warmest welcomes to **tomspicy**, our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/437/header.jpg?1)
+
+**tomspicy** is an electronic artist whose music career was kickstarted by passionate public reception to a meme track: the [fieryrage accordion](https://www.youtube.com/watch?v=yoV794KYq48) [hardcore remix](https://osu.ppy.sh/beatmapsets?q=%22fieryrage%20accordion%22)".
+
+Ever since the short tune's release in 2021, **tomspicy** has been dedicated to electronic music, primarily outputting drum and bass with occassionally dips into other electronic subgenres like hardcore and old school jungle. As of today, **tomspicy** is also dipping into our Featured Artist library!
```

Comment:
```suggestion
Ever since the short tune's release in 2021, **tomspicy** has been dedicated to electronic music, primarily outputting drum and bass with occasional dips into other electronic subgenres like hardcore and old school jungle. As of today, **tomspicy** is also dipping into our Featured Artist library!
```